srcpkgs/gopass/template

```diff
@@ -13,7 +13,7 @@ license="MIT"
 homepage="https://www.gopass.pw/"
 changelog="https://raw.githubusercontent.com/gopasspw/gopass/master/CHANGELOG.md"
 distfiles="https://github.com/gopasspw/gopass/archive/v${version}.tar.gz"
-checksum=f4bfe03bc26701baf411795faafe774aa6025bc6050ab3ebf7e931e5ff821961
+checksum=26ac90ef60d7f08e7d9361280d6457edb90642e821edba2c5e5cb0a09ca20b08
 
 if [ "$CROSS_BUILD" ]; then
```

Comment:
You should be able to remove this block if you switch to using `vtargetrun`, and the lines below could also use `vcompletion`. Please see `rustup` for an example.

Reply:
Got it, thanks!